bundles/net.tourbook/src/net/tourbook/tour/DialogSetTimeZone_WizardPage.java

```diff
@@ -15,6 +15,8 @@
  *******************************************************************************/
 package net.tourbook.tour;
 
+import static org.eclipse.swt.events.SelectionListener.widgetSelectedAdapter;
```

Comment:
I'm not using such static imports any more because when I change some code in this file and it contains errors and save it, this static import is auto removed and all dependencies in this file are causing errors. This happend to me for several times and since then I don't use these static imports.

You can keep it here for now but when I modify a file and is causing such an error, I remove the static import.

Reply:
Interesting, I have never experienced such an error.

I tend to prefer static adapters because they take less lines and are easier to read

Reply:
> I tend to prefer static adapters because they take less lines and are easier to read

I also would use it but not when I have to correct the code

In the video you can see that when forcing an error and then saving the code, the static import is removed.


https://github.com/mytourbook/mytourbook/assets/1283445/ca2e59fa-4115-4a56-83a0-bd84ee3bf5fa

These are my autosave actions

![image](https://github.com/mytourbook/mytourbook/assets/1283445/66bb2b35-07c7-4ecd-9920-d523784e280a)

Reply:
Interesting, here are my autosave actions but I don't see any obvious difference that would cause that:
But sometimes, Eclipse can be full of surprises that can't be explained

![image](https://github.com/mytourbook/mytourbook/assets/25235665/77006324-eb81-4b8e-8f5a-11b4b6b94c06)